package.json

```diff
@@ -13,29 +13,30 @@
     "type": "git"
   },
   "scripts": {
-    "check-types": "yarn check-types:build && yarn check-types:tests",
+    "check-types": "pnpm check-types:build && pnpm check-types:tests",
     "check-types:tests": "tsc -p tsconfig.test.json --noEmit",
-    "check-types:build": "lerna exec tsc -- --noEmit",
+    "check-types:build": "pnpm -r --parallel exec tsc --noEmit",
     "prettier": "prettier --ignore-path .gitignore '**/*.+(js|json|ts|md|yml|yaml)'",
-    "format": "yarn prettier --write",
-    "check-format": "yarn prettier --list-different",
-    "clean": "lerna run clean",
-    "build": "lerna run build",
+    "format": "pnpm prettier --write",
+    "check-format": "pnpm prettier --list-different",
+    "clean": "pnpm -r --parallel run clean",
+    "build": "pnpm -r --parallel run build",
     "test:unit": "jest --testPathIgnorePatterns 'e2e.test.ts$'",
     "test:e2e": "jest --testMatch '**/?(*.)e2e.test.ts'",
     "test": "jest",
     "lint": "eslint --ignore-path .gitignore .",
-    "validate": "yarn lint && yarn check-types && yarn check-format",
-    "run-mediator": "ts-node ./samples/mediator.ts",
-    "next-version-bump": "ts-node ./scripts/get-next-bump.ts"
+    "validate": "pnpm lint && pnpm check-types && pnpm check-format",
+    "run-mediator": "ts-node ./samples/mediator.ts"
   },
   "devDependencies": {
+    "@changesets/cli": "^2.27.5",
     "@hyperledger/aries-askar-nodejs": "^0.2.1",
+    "@jest/types": "^29.6.3",
     "@types/bn.js": "^5.1.5",
     "@types/cors": "^2.8.10",
     "@types/eslint": "^8.21.2",
     "@types/express": "^4.17.13",
-    "@types/jest": "^29.5.11",
+    "@types/jest": "^29.5.12",
```

Comment:
types/jest and jest/types are both required?

Reply:
yes, sadly. Was getting issues with one of them not being defined (I think this has to do with the strict dependency resolution from pnpm, as `@jest/types` was not needed before)